docker/Dockerfile

```diff
@@ -73,4 +80,27 @@ COPY run-all.sh .
 # hadolint ignore=SC2038,DL4006
 RUN find . -maxdepth 1 -type f | xargs dos2unix
 
+FROM redhat/ubi9-minimal:9.5-1742914212
```

Comment:
Upon taking another look, I discovered that `redhat/ubi9-micro` is the smallest image we can get. 

Fwiw, Alpine is still *way* smaller compared to any of the UBI images. Though I'd safer to go to a smaller UBI image for now and consider if we should try to further squeeze this aspect of the overall size :)